src/synchronicity/type_stubs.py

```diff
@@ -135,6 +135,8 @@ def _get_type_vars(typ, synchronizer, home_module):
         # The reason it cant be used directly is that this method should return the original type params
         # and not translated values
         if isinstance(typ, typing.ForwardRef):  # TypeVars wrap their arguments as ForwardRefs (sometimes?)
+            if hasattr(typ, "__forward_module__") and typ.__forward_module__ is not None:
```

Comment:
Could there be cases where `__forward_arg__` would still need evaluation here (i.e. cases that don't correspond to our synthetic ForwardRef insertions)? Maybe we should check `__forward_module__` vs `TYPE_CHECKING_OVERRIDES` at this point before returning (so we can still try to evaluate the string annotation if it's not one of these modules)